### DISCUSSION TITLE
The project got stuck at startup

### DISCUSSION THREAD

**Discussion by 18872297546** [webpack/webpack#18690] [Q&A]
hello everyone!!

I got stuck when using webpack-5 to start a project after the server started, without any error messages

Then I tried debugging and found that the program froze after entering a file at the beginning of a VM

Next, I added print information to the restore method in PackFileCacheStrategy.js and found that the program froze after a random module

``` javascript
restore(identifier, etag) {
  return this._getPack()
     .then(pack => {
        console.log('pack.requests.length >>>', pack.requests.length )
  })
}
```
As shown in the above code, when compiling to a certain file, it will suddenly get stuck, and this file is not fixed

I have tried clearing the cache, deleting node_modules, and reinstalling dependencies, but none of these issues have been resolved

This problem has been bothering me for several days, and I hope some experts can help give me some advice

Thank you very much. Great kindness and virtue will be remembered in my heart

webpack version:  5.72.0
Node.js version: 16.13.0
Operating System: windows10
Additional tools: pnpm@7.33.4, yarn@1.22.22

**Comment by alexander-akait** [comment]
Can you create reprducible test repo? Need investigate

**Reply by 18872297546** [reply]
Thank you very much for your reply, but this project is on the intranet and cannot be published on the public network

**Reply by 18872297546** [reply]
It's strange that at the beginning it was possible to run, but one day suddenly there was a problem

**Reply by alexander-akait** [reply]
Check your permissions on cache folder, also check you don't have special characters in path to cache, also can you show me a configuration file
